PhysicsTools/PatAlgos/python/slimming/lostTracks_cfi.py

```diff
@@ -10,8 +10,11 @@
     minPt = cms.double(0.95),	
     minHits = cms.uint32(8),	
     minPixelHits = cms.uint32(1),  
-    covarianceVersion = cms.int32(1), #so far: 0 is Phase0, 1 is Phase1   
+    covarianceVersion = cms.int32(0), #so far: 0 is Phase0, 1 is Phase1   
     covarianceSchema = cms.int32(0), #old miniaod like
     qualsToAutoAccept = cms.vstring("highPurity"),
     minPtToStoreProps = cms.double(0.95)	
 )
+from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
+phase1Pixel.toModify(lostTracks, covarianceVersion =1 )
+
```

Comment:
@slava77 can you comment if this is the right impl of an Era switch, otherwise please point me to some docs